packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
Please see original review comments in https://github.com/okta/okta-developer-docs/pull/168